Web.Client/Web.Client.csproj

```diff
@@ -34,7 +34,22 @@
 	</ItemGroup>
 
 	<ItemGroup>
+	  <_ContentIncludedByDefault Remove="Pages\Admin\GradeRegistrationCriteriaEdit.razor" />
+	  <_ContentIncludedByDefault Remove="Pages\Admin\GradeRegistrationCriteriaList.razor" />
 	  <_ContentIncludedByDefault Remove="Pages\Development\MartinFiala.razor" />
+	  <_ContentIncludedByDefault Remove="Pages\Electives\ElectiveProgressComponent.razor" />
+	</ItemGroup>
+
+	<ItemGroup>
+	  <UpToDateCheckInput Remove="Pages\Electives\ElectiveProgressComponent.razor" />
+	</ItemGroup>
+
+	<ItemGroup>
+	  <UpToDateCheckInput Remove="Pages\Admin\GradeRegistrationCriteriaList.razor" />
+	</ItemGroup>
+
+	<ItemGroup>
+	  <UpToDateCheckInput Remove="Pages\Admin\GradeRegistrationCriteriaEdit.razor" />
```

Comment:
Tohle začalo do .csproj nějak přidávat Visual Studio. Ještě jsem nepochopil za jakých okolností k tomu dojde, ale potřeba to není.